data/common.yaml

```diff
@@ -11,8 +11,10 @@ bacula::director::messages:
       catalog:      'all'
 bacula::director::postgresql::make_bacula_tables: ''
 bacula::director::packages: []
+bacula::director::ensure: 'present'
```

Comment:
Putting the default value in Hiera only makes sense when don't have the same default for all platforms (we fail to do this in the module ATM but I think I will tackle this one day).  Can you please update the PR to put the default in the manifest instead?  One key benefit is that the default value will be visible when we generate the REFERENCE file.

(same for `bacula::storage::ensure` and `bacula::client::ensure` bellow)

Reply:
Done. (Although I usually like to keep code and data strictly separated, with all default values in Hiera.)